Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -1368,7 +1467,7 @@ def fetch_incidents(client: AzureSentinelClient, last_run: dict, first_fetch_tim
 
     """
     # Get the last fetch details, if exist
-    limit = demisto.params().get("limit", DEFAULT_LIMIT)
+    limit = min(arg_to_number(demisto.params().get("limit")) or FETCH_MAX_LIMIT, FETCH_MAX_LIMIT)
```

Comment:
```suggestion
    limit = min(arg_to_number(demisto.params().get("limit")) , FETCH_MAX_LIMIT)
```
The param has default value

Reply:
This change will raise mypy error: 
Type "int | None" is not assignable to type "int" since min needs two int and arg_to_number return value is int|None

Reply:
So we can at list set the default value like this:
demisto.params().get("limit", FETCH_MAX_LIMIT)

Reply:
The mypy error is because arg_to_number return None | int and min method want to get an int - so we leave it as is and didn't change it after a discussion.